config/application.rb

```diff
@@ -1,25 +1,40 @@
-require File.expand_path('../../env', __FILE__)
```

Comment:
re-templated by `bin/rails app:updated`, retained the `require env.rb` on the first line.